src/components/main.tsx

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import classNames from 'classnames';
+import React from 'react';
+
+export default function Main({ todos }) {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+      {/* This is a completed todo */}
+
+      {todos.map(todo => {
+        return (
```

Comment:
```suggestion
      {todos.map(todo => (
```

can be without return